src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -457,7 +457,6 @@ get_service_config_from_resolver_result_locked(channel_data* chand) {
         grpc_uri* uri = grpc_uri_parse(server_uri, true);
         GPR_ASSERT(uri->path[0] != '\0');
         service_config_parsing_state parsing_state;
-        memset(&parsing_state, 0, sizeof(parsing_state));
```

Comment:
I don't see a replacement for this one ?

Reply:
Everything in this object is set below